tests/APIv2/UsersTest.php

```diff
@@ -129,6 +129,37 @@ public function testDeletePhoto() {
         $this->assertStringEndsWith('/applications/dashboard/design/images/defaulticon.png', $user['photoUrl']);
     }
 
+    /**
+     * Test confirm email is successful
```

Comment:
Make sure your descriptions end in a full stop.

Reply:
resolved